src/main/java/org/team1540/robot2024/Robot.java

```diff
@@ -176,19 +169,27 @@ public void autonomousInit() {
 
     }
 
+    @Override
+    public void autonomousExit() {
+        robotContainer.enableBrakeMode(true);
+    }
```

Comment:
SINCE WHEN WAS THIS A THING?????? Also does DriverStation.isEnabled() report false by the time this is called?

Reply:
uh always and looks like it